moroScript.jsx

```diff
@@ -115,6 +117,23 @@
         return rtnWord;
     }
 
+    //Process dict with count to sorted dict without count value
+    function sortAndRemoveCount(dict) {
+        var toRtn = JSON.parse(JSON.stringify(dict));
```

Comment:
Yes. That is to make a new copy of the object. We keep the old object with the count property in case we need it in the future (to show the count of each moroword, etc.). I have not heard of lodash before. I will look into it. However, any method of creating a new copy is fine.